### ISSUE TITLE
Device not in sync for TOTP?

### ISSUE THREAD

**Issue by GuillaumeHemmen** [mooltipass/moolticute#849]
### Expected behaviour
- The device is in synch and gives the correct TOTP.

### Actual behaviour
- The device is not in synch and gives an incorrect TOTP.

### Step by step guide to reproducing the problem
Prerequisite:
- Any TOTP Software (like google auth, authy, etc) or online TOTP demo (like https://totp.danhersam.com).
- TOTP token: JBSWY3DPEHPK3PXP (randomly generated)
- Have the TOTP set up in the TOTP software/demo to check the TOTP values.

Reproducibility:
- Create a login (either by adding it when detected with the browser extension or add it through moolticute).
- Open moolticute credential management mode and add the TOTP token to this credential.
- Exit moolticute credential management mode.
- Navigate with your device to display the TOTP.
- The TOTP doesn't correspond to the other generators.

Results:
-  It seems that the device is ahead of ±30 seconds concerning the TOTP code generation. As soon as the code rotate, on both the device and the google auth I can see a new value on the device and the "old" value on google auth (e.g., first token rotation: TOTP is 111111 on the device and 123456 on google auth. next token rotation: TOTP is 999999 on the device and 111111 on the google auth)

### Moolticute Version
- v0.46.3

### Operating System
Tested on: 
- Windows 10
- macOS  

### Mooltipass Device
- The Mooltipass Mini BLE

#### Additional info
- macOS dev logs: [minible_dev_log.txt](https://github.com/mooltipass/moolticute/files/6650045/minible_dev_log.txt)
- Windows dev logs: [minible_dev_log_win.txt](https://github.com/mooltipass/moolticute/files/6650074/minible_dev_log_win.txt)

**Comment by limpkin** [comment]
Hello!
Does this always happen on your computer? could you try on another one? I'm wondering if there's something special on your computer preventing us from getting the correct time.

**Comment by limpkin** [comment]
re-reading this issue it seems my idea is non-relevant...
However, could you double check that on actual services your attempts get denied? In theory our device should be compliant with this +-30 seconds window.
In more details, on our device we do have a calibration routine to reduce that time difference to a minimum: the more you let your device connected to your computer, the more calibrated it will become.

**Comment by GuillaumeHemmen** [comment]
Hello @limpkin,

I have used my BLE for a few days now (USB/BT) without the TOTP feature to let time for the time sync. This weekend I have started to add a few TOTP credentials, and so far, so good. Since the issue is not reproducible anymore, I will close this issue!

**Comment by TheGreatBeginner777** [comment]
This one is driving me crazy so I had to reopen this subject. Mini BLE TOTP vs Yubico TOTP. FYI: TOTP from Yubico is the correct one and accepted by services. From what I can see Mini BLE is ahead 30 seconds. Also I checked and 30 second interval between two is the same (mening they have the same countdown...). Example from looking at particular service where 2FA is enabled is shown below.
Mini BLE | Yubico
190131 | 742487
347449 | 190131
587989 | 347449
814106 | 587989
....
@limpkin "the more you let your device connected to your computer, the more calibrated it will become."
How to achieve this? Mine BLE is always connected to computer but it goes to sleep after some period of time. Does that fact (going to sleep) affects calibration routine?
